docs/cli.md

```
@@ -1019,6 +1021,39 @@ openspec schema fork <source> [name] [options]
 openspec schema fork spec-driven my-workflow
 ```
 
+When the source has an active layered user override, the fork materializes the effective schema and effective user/package templates into a self-contained project bundle.
+
+---
+
+### `openspec schema override`
+
+Create a layered user override for a packaged schema. Unlike `schema fork`, this command does not copy `schema.yaml` or its templates; packaged updates remain active for fields and templates you do not override.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)